### DISCUSSION TITLE
FULLNODE - KEYSTORE FOLDER

### DISCUSSION THREAD

**Discussion by vinatechpro** [availproject/avail#426] [Q&A]
A file in the folder named keystore is the seed of the wallet which is used by the node? 
For what purpose is it used?
`~/avail/output/chains/avail_goldberg_testnet/keystore`

**Comment by adheretoself** [comment]
I have also paid attention to this matter and am equally curious.

**Reply by 0xKaytee** [reply]
I found out the same and I would like to know this too.

**Comment by Leouarz** [comment]
It's generally used to sign stuff as a validator / authority, you can learn more here : https://docs.substrate.io/tutorials/build-a-blockchain/add-trusted-nodes/

Especially in the `Add keys to the keystore` section
